src/services/evacuationPathfinder.ts

```
@@ -0,0 +1,43 @@
+import { SpatialPoint, VenueCanvasLayout, EmergencyExit } from '../../shared/schemas/evacuationAR';
```

ESLint

[error] Replace `'../../shared/schemas/evacuationAR'` with `"../../shared/schemas/evacuationAR"`

(prettier/prettier)